### DISCUSSION TITLE
Announce the section title when changing sections using Alt+Down/UpArrow

### DISCUSSION THREAD

**Discussion by cary-rowen** [mush42/bookworm#120] [General]
Hi @mush42 

Press Alt + Up/Down Arrow in the text of the document to switch chapters of the document, NVDA should report the current chapter timely.
I don't know if this is an NVDA bug or if Bookworm is not sending notifications.

Thanks

**Comment by mush42** [comment]
Probably it is a bookworm bug.

**Comment by cary-rowen** [comment]
Ok, do I need to create a corresponding Issue so you can fix it?
